### PR TITLE
Shorthand formatting step

### PR DESCRIPTION
**Pull request by NunoSempere**
=mm(-10 to 10, ...) -> =mm(normal(0, 6), ...)

### CODE REVIEW

File: packages/guesstimator/formatter/formatters/lib.js [hunk 1]
```diff
@@ -74,8 +76,9 @@ export function shorthandIntoLognormalFormattingStep(text){
       
         let low = arrayLowHigh[0]
         let high= arrayLowHigh[1]
-
-        return distributionUpToIntoLognormal(low, high)
+        
+        result = (getGuesstimateType("unknown yet", [low])=='LOGNORMAL') ? distributionUpToIntoLognormal(low, high) : distributionUpToIntoNormal(low, high)
```
Comment (OAGr) [inline packages/guesstimator/formatter/formatters/lib.js]: I imagine there must be some cleaner way of doing this

File: packages/guesstimator/lib/distributionMath.js [hunk 2]
```diff
@@ -10,3 +10,14 @@ export function distributionUpToIntoLognormal(low, high){
...
     return `lognormal(${mean},${stdev})`
 }
+
+// The left endpoint marks 0.05% on the CDF, the right 0.95%.
+export function distributionUpToIntoNormal(low, high){
+    
+    let mean = math.mean(high, low)
+    let stdev = (high - mean) / 1.645
+    return `normal(${mean},${stdev})`
+
```
Comment (OAGr) [inline packages/guesstimator/lib/distributionMath.js]: why the spaces at the top, and especially the bottom? Also, underneath this function. Please delete.

File: packages/guesstimator/formatter/formatters/lib.js (hunk 1 above)
Comment (NunoSempere) [reply packages/guesstimator/formatter/formatters/lib.js]: Yes, but not if you want to reuse code.

File: packages/guesstimator/lib/distributionMath.js (hunk 2 above)
Comment (NunoSempere) [reply packages/guesstimator/lib/distributionMath.js]: Done

**Pull request merged**